crates/aranya-daemon/src/sync/prot.rs

```diff
@@ -0,0 +1,13 @@
+//! Aranya syncer supported protocols.
+//! New protocols must be added to the end of this list since protocol types can never change.
+
+/// 0 indicates an error.
+pub const PROTOCOL_ERR: u8 = 0;
+
+#[repr(u8)]
+#[non_exhaustive]
+/// Protocols supported by the syncer.
+pub enum SyncProtocol {
+    /// Version 1.
+    V1 = 1,
+}
```

Comment:
Could include the error in the enum definition:
```suggestion
#[repr(u8)]
#[non_exhaustive]
/// Protocols supported by the syncer.
pub enum SyncProtocol {
    /// Indicates an error.
    PROTOCOL_ERR = 0,
    /// Version 1.
    V1 = 1,
}
```

Reply:
I don't understand why we need `PROTOCOL_ERR` at all tbh